user-guide/Advanced_Functionality/Databases/Databases_about.md

```diff
@@ -6,7 +6,7 @@ uid: Databases_about
 
```

Comment:
I will restructure/rewrite this *About* page, once we are completely happy with the Databases overhaul and new structure.